superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -146,9 +153,9 @@ def get_recipe_text(self, distributor, license_text):
             ret += 'LICENSE = "'
             ret += ' & '.join([get_license(l) for l in self.license]) + '"\n'
         ret += 'LIC_FILES_CHKSUM = "file://package.xml;beginline='
-        ret += str(self.license_line)
+        ret += str(self.license_line_begin)
         ret += ';endline='
-        ret += str(self.license_line)
+        ret += str(self.license_line_end)
```

Comment:
I think we are solving a different problem here -- please don't include such a fix in this PR.

Reply:
Done